app/controllers/api/health.php

```diff
@@ -798,15 +798,15 @@
         $response->dynamic(new Document([ 'size' => $size ]), Response::MODEL_HEALTH_QUEUE);
     });
 
-App::get('/v1/health/queue/usage-dump')
+App::get('/v1/health/queue/stats-usage-dump')
```

Comment:
Let's make sure to update any health checks.

Reply:
added issue to track it